index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+    <!-- Start of the header -->
+    <header>
+        <div class="center">
+        <nav>
+            <ul>
+                <li><span><a href="./about.html" >About Me</a></span></li>
+                <li><span><a href="./portfolio.html">Soriano's Portfolio</a></span></li>
+                <!-- <li><span><a href="mailto:dsorin@proton.me" class="button">Contact</a></span></li> -->
+            </ul>
+        </nav>
+    </div>
+</header>
+    <<body>
```

Comment:
We don't want to use more than one `body` tag per document. I'd suggest running this HTML through the validator https://validator.w3.org/#validate_by_input to see what other potential issues might be present.